README.md

```diff
@@ -2541,6 +2543,54 @@ Gets the value at path of object.
 
   **[⬆ back to top](#quick-links)**
 
+### _.invert
+
+Creates an object composed of the inverted keys and values of object. If object contains duplicate values, subsequent values overwrite property assignments of previous values.
+
+  ```js
+  var object = { 'a': 1, 'b': 2, 'c': 1 };
+
+  // Underscore/Lodash
+  var result = _.invert(object);
+  console.log(result)
+  // output: { '1': 'c', '2': 'b' }
+
+  // Native (IE6)
+  function invert(object) {
+    var obj = {};
+    for (var key in object) {
+      if (object.hasOwnProperty(key)) {
+        obj[object[key]] = key;
+      }
+    }
+    return obj;
+  }
+  var result = invert(object);
+  console.log(result)
+  // output: { '1': 'c', '2': 'b' }
+
+  // Native (IE not supported)
+  const invert = object => Object.entries(object)
+    .reduce((acc, current) => {
+      acc[current[1]] = current[0];
+      return acc;
+    }, {}
+  );
```

Comment:
```suggestion
const invert = (object) =>
  Object.fromEntries(
    Object.entries(object)
      .map(([k, v]) => [v, k])
    );
```

Even simpler using `fromEntries` from ES2019

Reply:
`Object.entries` already creates garbage/GC and then even do that for every key? You are better off just reversing it:

```js
object = {'key x': 'val x', 'key y': 'val y', 'key z': 'val z'}
Object.fromEntries(Object.entries(object).map(_ => _.reverse()))
```

But then again, why use any function that creates GC if it is possible to not produce garbage at all? After all it's simply:

```js
ret = {};
for (const k in object) {
  const v  = object[k];
  ret[v] = k;
}
```